README.md

```diff
@@ -139,7 +139,7 @@ And it easy to extend this typeclass with your own classes as well:
 ```
 
 That's how simple, safe, and powerful typeclasses are!
-Make sure to [check out our docs](https://github.com/dry-python/classes) to learn more.
+Make sure to [check out our full docs](https://github.com/dry-python/classes) to learn more.
```

Comment:
Just a question, why `full`? Hahahaha

Reply:
I had a warning that we have two similar anchors!